cmd/rookflex/cmd/mount.go

```diff
@@ -140,8 +140,8 @@ func mountDevice(client *rpc.Client, mounter *k8smount.SafeFormatAndMount, devic
 	// formatted even though one does exist. K8s then proceeds to format the volume which would result in data loss. This logging is an attempt
 	// to understand why k8s believes the volume is not formatted. See https://github.com/rook/rook/issues/1553
 	log(client, fmt.Sprintf("Testing to see if device %s needs formatting...", devicePath), false)
-	executor := &exec.CommandExecutor{}
-	output, err := executor.ExecuteCommandWithOutput(false, "", "blkid", "-p", "-s", "TYPE", "-s", "PTTYPE", "-o", "export", devicePath)
+	cmd := exec.Command("blkid", "-p", "-s", "TYPE", "-s", "PTTYPE", "-o", "export", devicePath)
```

Comment:
Why not change the actual exec.go code to have a "ExecutedCommandWithOutputNoLog" or a "don't log" parameter?

Reply:
i'm not sure it's worth changing the exec package. I'd recommend we just revert the change since we expect it to be fixed by updating the k8s client.